docs/reference/server-configuration.md

```diff
@@ -487,6 +487,20 @@ the pull query REST endpoint (by including it in the request e.g: `"streamsPrope
 By default, any amount of lag is allowed. For using this functionality, the server must be configured with `ksql.heartbeat.enable=true` and 
 `ksql.lag.reporting.enable=true`, so the servers can exchange lag information between themselves ahead of time, to validate pull queries against the allowed lag.
 
+## `ksql.query.pull.table.scan.enabled`
+
+**Per query:** yes
```

Comment:
is there a possibility to set this at the server level? some deployments might want to prevent table scans altogether

EDIT: i notice in the code that you can (but you can't override the server default to enable them). probably worth calling out in the docs